plugin/extkubernetes/README.md

```diff
@@ -0,0 +1,97 @@
+# extkubernetes
+
+## Name
+
+*extkubernetes* - enables reading external zone data from a Kubernetes cluster for Load Balancer IPs.
+
+## Description
+
+Creates A/AAAA, SRV, and PTR records for the External IPs of each LoadBalancer type Service in a Kubernetes cluster.
+
+
+This plugin can only be used once per Server Block.
+
+## Syntax
+
+~~~
+extkubernetes [ZONES...]
+~~~
+
+With only the directive specified, the *extkubernetes* plugin will default to the zones specified in
+the server's block. It will handle all queries in those zones and connect to Kubernetes in-cluster.
+If **ZONES** is used, it specifies all the zones the plugin should be authoritative for.
+
+```
+extkubernetes [ZONES...] {
+    resyncperiod DURATION
+    endpoint URL [URL...]
+    tls CERT KEY CACERT
+    kubeconfig KUBECONFIG CONTEXT
+    namespaces NAMESPACE...
+    labels EXPRESSION
+    ttl TTL
+    transfer to ADDRESS...
+    fallthrough [ZONES...]
+}
+```
+
+* `resyncperiod` specifies the Kubernetes API refresh period to be **DURATION**.
```

Comment:
Just reference the kubernetes plugin here? Removes the need to duplicate all the info here.